File: Unidad 3/Java/Programa16U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa16u3;

import java.util.Scanner;
public class Programa16U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int  c= 1, horas,pago,sueldo;
        System.out.println("Determinar el sueldo semanal de horas trabajadas");
        while(c <= 20){
            System.out.println("Introduce las horas trabajadas ");
            horas= sc.nextInt();
            System.out.println("Introduce el sueldo por hora ");
            pago= sc.nextInt();
            sueldo = horas * pago;
            System.out.println("El sueldo del trabajador es "+sueldo);
            c++;
        }
    }
}

File: Unidad 3/C#/Programa16U3.cs
// See https://aka.ms/new-console-template for more information
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Programa16U3
    {
        static void Main(string[] args)
        {
            int c = 1, horas, pago, sueldo;
            Console.WriteLine("Determinar el sueldo semanal de horas trabajadas");
            while (c <= 20)
            {
                Console.WriteLine("Introduce las horas trabajadas ");
                horas = Convert.ToInt32(Console.ReadLine());
                Console.WriteLine("Introduce el sueldo por hora ");
                pago = Convert.ToInt32(Console.ReadLine());
                sueldo = horas * pago;
                Console.WriteLine("El sueldo del trabajador es " + sueldo);
                c++;
            }
        }
    }
}